chart/values.yaml

```diff
@@ -46,6 +46,9 @@ podAnnotations: {}
 
 podSecurityContext:
   runAsNonRoot: true
+  fsGroup: 65532
+  runAsUser: 65532
+  runAsGroup: 65532
 
```

Comment:
why do we need this ?

Reply:
it was to match the user / group defined in the Dockerfile of the Operator here: https://github.com/cyril-corbon/druid-operator/blob/master/Dockerfile#L31 
I also note this : 
`set user / group / fsuser or the operator crash in 1.26`
but I cannot reproduce it anymore

Reply:
IMHO It's a good idea to set theses values by default, both containers have the same uid/gid
```
docker history gcr.io/kubebuilder/kube-rbac-proxy:v0.13.1
IMAGE          CREATED        CREATED BY                                      SIZE      COMMENT
60cb20b77a0f   9 months ago   /bin/sh -c #(nop)  ENTRYPOINT ["/usr/local/b…   0B
<missing>      9 months ago   /bin/sh -c #(nop)  USER 65532:65532             0B
```
I can remove it if you want or if you think it's not useful 😄

Reply:
make sense. 
its only related to kube rbac proxy container right ? not the druid-operator ?

Reply:
it's related to both containers. its not an issue as they share the same uid / gid